app-text/al-anvar/additional-files/al-anvar.rdef.in

```diff
@@ -0,0 +1,42 @@
+
+resource app_flags B_MULTIPLE_LAUNCH;
+
+resource app_version {
+	major  = @MAJOR@,
+	middle = @MIDDLE@,
+	minor  = @MINOR@,
+
+	variety = B_APPV_FINAL,
+	internal = 0,
+	short_info = "Al-Anvar",
+	long_info = "@LONG_INFO@"
+};
+
+resource app_signature "@APP_SIGNATURE@";
+
+resource file_types message {
+	"types" = "application/x-nvr"
+};
+
+resource vector_icon {
+	$"6E6369660B04018003010000020006023CC800389C28B89C283CC80049DBD7C6"
+	$"AAE100BAECFFFF008FFF020006023CC800389C28B89C283CC80049DBD7C6AAE1"
+	$"0075C2FFFF02366D020016023C5D733B1C57BB1C573C5D734BAB5648A8587157"
+	$"26B502001602BC54813B3720BB3720BC548149B0984B053BFFBE00FF02000602"
+	$"3CC800389C28B89C283CC80049DBD7C6AAE1000000FFFFF2F2F202001602BC62"
+	$"2BBB97633B9D05BC65E847C6844C30460057FFF402001602BBBEF1BA971F3AB9"
+	$"A5BBF0DE482F564BDE8F0057FFF402001602BC591EBB29993B2999BC591E481A"
+	$"B84BD27A0057FFF405000B0A05445D495D603E5A3B5B3F0607A62B223C49445B"
+	$"5A3E5D375D3A5D345A3139250A04243B444B5A313A260A0433323542483D482D"
+	$"0606AE0B444B4652464E4656445B5A3E5C375C3B5C345A310605E602223C4944"
+	$"5B46534657464F444B0802425625470802B642C1E6C0BCC70708022540434E02"
+	$"21BF1EBEE9BF1EBEE9BF1EBEE9BF25BF26BF25BF26BF25BF7EBECFC020BF09BF"
+	$"D1BE99C06FBDF9C0CCBE4FC0A7BDA0C0F1BD24C0C5BD5AC0EFBCEBC09EBCD0C0"
+	$"01BCD0C05ABCD0BF62BD8DBE1EBD0FBEC1BE0ABD7EBF7CBCACBEAFBD02C039BC"
+	$"5EC14EBC85C0D5BC51C1C5BCB9C203BDB4C203BD1DC203BE1BC1B2BF14C1E8BE"
+	$"90C17ABF95C041C133C101C048BF83C21DBEE3C323BF0DC2C1BEB5C385BE9FC4"
+	$"25BE9FC3D9BE9FC462BED1C4AABEB2C48C3EC4C9BF49C4BBBF1AC4CFBF74C4A8"
+	$"BFD6C423BFA5C476BFD6C423C005C42CC005C42CBFF3C47EBFA5C504BFD6C4C3"
+	$"BF73C542BF03C587BF40C56EBEBCC5A4BE57C57FBE81C5A1BE2AC558BE16C4B7"
+	$"BE16C517BE16C45DBE5AC389BE2BC3F8BE88C318BF7CC1A2BEE7C274C00DC0CE"
+	};
```

Comment:
```diff
-	};
+};
```
The above tab is not needed, but the main issue is that the vector icon is not being shown.
If I replace it with a copy of another icon from a recipe which has a valid icon then I can see the icon in the app.
So this means there is something wrong with the above bloc.

BTW, the app builds fine on both x86_64 and x86 (secondary arch).

Reply:
@mazbrili, if you can't update the recipe but can copy/paste (in a message below this one) the correct contents of this bloc:
```
resource vector_icon {
...
}
```
then I'll update your PR for you and merge it if the icon is ok :)

Remark: I wrote _correct_ because it seems the bloc we have right now is invalid. Maybe you made a tiny mistake in a copy/paste.

**_To all:_** I don't know how popular this app will be among the current users of Haiku, but I guess it might bring new users and maybe contributors too :)
So unless someone has a strong argument against merging this PR I believe it will be fine to do so once the issue about the vector icon is fixed.

Reply:
For some reason if I install the package it doesn't show up in $appsDir nor in the Deskbar, if I try to launch it from the bin folder in the source tree I'm getting missing libraries for Qt5 ...

Reply:
@Begasus, I guess you have built for x86 (secondary arch.)
Something is wrong on my x86_gcc2 hybrid system with Qt apps: I can start them but they don't show up. On x86_64 the app shows up correctly, even if I run it without access to the internet.

Does `/system/apps/Al-Anvar` exist? (It does for me, both on x86_gcc2 and x86_64.)

Reply:
@fbrosson app doesn't exists in the $appsDir after install (content looks ok in HaikuDepot) and yes build as _x86 on gcc2h

Reply:
Are you sure your recipe is in sync with this PR?
```bash
$ md5sum al_anvar-0.5.0.recipe 
8ec6c1a83e0e3d3c90a9619d68623d61  al_anvar-0.5.0.recipe
```

Reply:
@Begasus, do you get the same output as me?
```bash
$ package list -p al_anvar_x86-0.5.0-1-x86_gcc2.hpkg | grep -i al-anvar
apps/Al-Anvar
data/deskbar/menu/Applications/Al-Anvar
data/mime_db/application/x-vnd.al-anvar
```

Reply:
seemed I had another outcome fom md5sum and the package list, after checking all should be ok (md5sum + package list), but still nothing seems to install ...

Reply:
I always use `pkgman install ...` to install a package. I know copying the hpkg also works, but you don't get enough feedback if something goes wrong.
Were you trying to install by just copying (or moving) the hpkg to `/system/packages/` ?

Reply:
Strangely enough after a reboot all was there, and running ok :)